gateways/authorizenet/types.go

```diff
@@ -152,6 +157,15 @@ type Customer struct {
 // Payment specifies the credit card to be authorized (only payment option for now)
 type Payment struct {
 	CreditCard CreditCard `json:"creditCard"`
+	OpaqueData OpaqueData `json:"opaqueData"`
```

Comment:
json order matters for authnet, lets just confirm that this is the correct order.

Reply:
OpaqueData shouldn't be there when CreditCard is there